### PR TITLE
op-guide: refine Ansible deployment of TiDB

### PR DESCRIPTION
**Pull request by lilin90**
@LinuxGit @CaitinChen PTAL

This PR only covers part of https://github.com/pingcap/docs-cn/pull/766 and https://github.com/pingcap/docs-cn/pull/772, because https://github.com/pingcap/docs-cn/pull/766 involves too many updates. I'll update them in several PRs.

FYI. I'll add the related links to the beginning part of this document when other documents are ready.

### CODE REVIEW

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -476,198 +583,83 @@ Edit the `inventory.ini` file and add the following host variable after the IP o
...
-172.16.10.62
-
-[all:vars]
-username = tidb
+# process supervision, [systemd, supervise]
+process_supervision = systemd
 ```
 
-Run the following command, and enter the `root` password of the deployment target machine as prompted:
+For versions earlier than TiDB 1.0.4, the TiDB-Ansible supervision method of a process is `supervise` by default. The previously installed cluster can remain the same. If you need to change the supervision method to `systemd`, close the cluster and run the following command:
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: stop the cluster

File: op-guide/ansible-deployment.md [hunk 2]
```diff
@@ -5,104 +5,282 @@ category: operations
...
+Make sure you have logged in to the Control Machine using the `tidb` user account, run the following command, and input the `root` password of your target machines as prompted:
+
+```bash
+$ cd /home/tidb/tidb-ansible
+$ ansible-playbook -i hosts.ini deploy_ntp.yml -k
+```
+
+The NTP service is installed and started using the software the comes with the system on the target machines. The default NTP server list in the installation package is used. The related `server` parameter is in the `/etc/ntp.conf` configuration file.
+
+To make the NTP service start synchronizing as soon as possible, the system executes `ntpdate` one time on `ntp_server` in the `hosts.ini` file. The default server is `pool.ntp.org`, and you can also replace it with your NTP server.
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: the system executes `ntpdate` sets the local date and time by polling `ntp_server` in the `hosts.ini` file.

File: op-guide/ansible-deployment.md [hunk 3]
```diff
@@ -5,104 +5,282 @@ category: operations
...
+> **Note:** If the time and time zone of all your target machines are same, the NTP service is on and is normally synchronizing time, you can ignore this step. See [How to check whether the NTP service is normal](#how-to-check-whether-the-ntp-service-is-normal).
+
+Make sure you have logged in to the Control Machine using the `tidb` user account, run the following command, and input the `root` password of your target machines as prompted:
+
+```bash
+$ cd /home/tidb/tidb-ansible
+$ ansible-playbook -i hosts.ini deploy_ntp.yml -k
+```
+
+The NTP service is installed and started using the software the comes with the system on the target machines. The default NTP server list in the installation package is used. The related `server` parameter is in the `/etc/ntp.conf` configuration file.
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: using the software repository that comes with the system

**Comment by lilin90** [comment]
@CaitinChen PTAL

**Pull request merged**